### PR TITLE
NEXT: Role count and average ilvl added to player selection page

### LINKED ISSUE #2961: Add role counts to raid composition page
image

| Tanks | Healers | Melee DPS | Ranged DPS |
| --|--|--|-- |
| 2 | 3 | 5 | 10 |

### PR DESCRIPTION
**Pull request by florianschut**
image

Fixes #2961 
Added composition details including role counts and average ilvl to the Player selection as requested in #2961 and #2962

### CODE REVIEW

**Comment by florianschut** [comment]
Added ilvl to the overview and moved the code to a new file to keep PlayerLoader readable

File: .env.local.example [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# Copy this file and rename it to `.env.local` to configure your own WCL API key. You can get your own API key from the bottom of the Warcraft Logs settings: https://www.warcraftlogs.com/profile
```
Comment (MartijnHols) [inline .env.local.example]: Please revert this file so it stays in tact for others

File: src/interface/report/PlayerLoader.js [hunk 2]
```diff
@@ -153,8 +186,21 @@ class PlayerLoader extends React.PureComponent {
                 </Link>
               </Tooltip>
             </div>
-            <h1 style={{ lineHeight: 1.4, margin: 0 }}><Trans>Player selection</Trans></h1>
-            <small style={{ marginTop: -5 }}><Trans>Select the player you wish to analyze.</Trans></small>
+            <div className="flex wrapable" style={{ marginBottom: 15 }}>
+              <div className="flex-main" style={{ position: 'relative' }}>
```
Comment (MartijnHols) [inline src/interface/report/PlayerLoader.js]:
```suggestion
              <div className="flex-main">
```

Doesn't seem used anywhere

File: src/interface/report/PlayerLoader.js [hunk 3]
```diff
@@ -153,8 +186,21 @@ class PlayerLoader extends React.PureComponent {
...
+                <h1 style={{ lineHeight: 1.4, margin: 0 }}><Trans>Player selection</Trans></h1>
+                <small style={{ marginTop: -5 }}><Trans>Select the player you wish to analyze.</Trans></small><br />
```
Comment (MartijnHols) [inline src/interface/report/PlayerLoader.js]:
```suggestion
                <small style={{ marginTop: -5 }}><Trans>Select the player you wish to analyze.</Trans></small>
```

File: src/interface/report/PlayerSelection/PlayerSelection.scss [hunk 4]
```diff
@@ -1,5 +1,35 @@
 @import 'interface/layout/Theme.scss';
 
+.raid-composition{
+  align-content:"center";
```
Comment (MartijnHols) [inline src/interface/report/PlayerSelection/PlayerSelection.scss]:
```suggestion
  align-content: center;
```

**Comment by florianschut** [comment]
Thanks for the feedback! The requested changes are made.

**Pull request merged**